src/components/itemPicker/CItemPicker.vue

```diff
@@ -0,0 +1,468 @@
+<template>
+  <div
+    class="overflow-hidden d-flex flex-column w-100 vh-100"
+  >
+    <b-card
+      no-body
+      :class="[!emptyColumnsHeight ? 'h-100' : '']"
```

Comment:
wouldn't class="h-100" also work? Or does emptyColumnsHeight change how it looks?